[2022-02-09 09:00:47,366] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:00:47,396] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:00:47,396] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:00:47,396] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:00:47,397] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:00:47,683] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-05-01 23:00:00+00:00
[2022-02-09 09:00:47,690] {standard_task_runner.py:52} INFO - Started process 456 to run task
[2022-02-09 09:00:47,698] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-05-01T23:00:00+00:00', '--job-id', '499', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpx7cqgyvg', '--error-file', '/tmp/tmp6ttee168']
[2022-02-09 09:00:47,702] {standard_task_runner.py:77} INFO - Job 499: Subtask local_to_gcs_task
[2022-02-09 09:00:48,191] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-05-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 09:00:48,548] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:00:48,689] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-05-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-01T23:00:00+00:00
[2022-02-09 09:01:00,768] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:01:00,783] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20210501T230000, start_date=20220209T090047, end_date=20220209T090100
[2022-02-09 09:01:00,819] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:01:00,868] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:16:39,788] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:16:39,801] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:16:39,801] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:16:39,801] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:16:39,802] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:16:39,816] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-05-01 23:00:00+00:00
[2022-02-09 09:16:39,822] {standard_task_runner.py:52} INFO - Started process 623 to run task
[2022-02-09 09:16:39,829] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-05-01T23:00:00+00:00', '--job-id', '604', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpegcova_w', '--error-file', '/tmp/tmp9p1u4ztq']
[2022-02-09 09:16:39,830] {standard_task_runner.py:77} INFO - Job 604: Subtask local_to_gcs_task
[2022-02-09 09:16:39,994] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-05-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:16:40,092] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:16:40,124] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-05-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-01T23:00:00+00:00
[2022-02-09 09:16:50,872] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:16:50,887] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20210501T230000, start_date=20220209T091639, end_date=20220209T091650
[2022-02-09 09:16:50,916] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:16:50,960] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
